src/main/java/io/cdap/cdap/metrics/process/gcp/GoogleCloudMonitoringUtil.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.metrics.process.gcp;
+
+import com.google.api.Metric;
+import com.google.api.MonitoredResource;
+import com.google.cloud.monitoring.v3.MetricServiceClient;
+import com.google.common.collect.Lists;
+import com.google.monitoring.v3.CreateTimeSeriesRequest;
+import com.google.monitoring.v3.Point;
+import com.google.monitoring.v3.ProjectName;
+import com.google.monitoring.v3.TimeInterval;
+import com.google.monitoring.v3.TimeSeries;
+import com.google.monitoring.v3.TypedValue;
+import com.google.protobuf.util.Timestamps;
+import io.cdap.cdap.api.metrics.MetricType;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Util class for GoogleCloudMonitoring
+ */
+public final class GoogleCloudMonitoringUtil {
+
+  private GoogleCloudMonitoringUtil() {
+
+  }
+
+  /**
```

Comment:
Give more/indepth context/details on  what the method does and how you expect caller to use it as part of the context, instead of repeating what the function name and variable name already tell you.

Reply:
Done.